tests/test_fixerio.py

```diff
@@ -0,0 +1,10 @@
+""" Tests for fixerio module """
+import logging
+from finance_quote import App
+
+def test_if_alive():
+    app = App()
+    result = app.fixerio("eur", "usd")
```

Comment:
this looks more an integration test than a unit test as it tests a lot of layers in the package (App, FixerIo, the caching, etc)

Reply:
Correct. That's why it is useful. :)

Reply:
We can have such tests but we must have unit tests ;-)

Reply:
hi @MisterY , could you add least one test that check for correctness of the call (checking the quote.value, quote.datetime, etc) for fixerio ? the current test is testing there is not syntax/runtime error not much more...